### DISCUSSION TITLE
Forbid to place backpack item inside another backpack

### DISCUSSION THREAD

**Discussion by koenigstag** [P3pp3rF1y/SophisticatedBackpacks#924] [Ideas]
While configuring my own modpack I've noticed that backpack can be placed inside one another, which makes them an infinite storage. Didn't check if second level of storage recursion can be done ingame, but I'm assuming it is in theory.

Suggestion is to add config that disables such abuse.

P.S. After investigating the source code I've found a server config option `canBePlacedInContainerItems`. Adding this to config while server is running results to correction of config and deletion of this property. I'm using sophisticated backpacks version 3.18.52.846. Maybe an upgrade to newer version will help.

**Comment by P3pp3rF1y** [comment]
if  you mean inside of sophisticated backpack that can only be done with inception upgrade in place if you don't like that you can just disable the upgrade in config. 

Not sure what you mean by adding the property in config, it should already be there (in `sophisticatedbackpacks-server.toml` and it's actually set to false by default so backpacks by default can't go inside other container items. But if that actually works that means the other backpack doesn't check for this property and you should really report that issue to them.

**Comment by koenigstag** [comment]
After an upgrade to version 3.19 config works as intended.

**Comment by koenigstag** [comment]
But I have a question - is it possible to have another config option like `canBePlacedInContainers` (inside chests, furnaces, etc.) ?

**Reply by P3pp3rF1y** [reply]
that would require special code to be inserted into vanilla implementation and still wouldn't solve what you are trying to do for any other modded inventory so in general I would say no. Also I don't see a good reason to do that as this storage should be stationery and if it can be moved it is with tapes or similar so you would want to limit those items in your modpack to prevent someone moving backpacks in a chest
